frontend/src/Calendar/iCal/CalendarLinkModalContent.js

```diff
@@ -115,70 +116,70 @@ class CalendarLinkModalContent extends Component {
     return (
       <ModalContent onModalClose={onModalClose}>
         <ModalHeader>
-          Sonarr Calendar Feed
+          {translate('CalendarFeed', { appName: 'Sonarr' })}
         </ModalHeader>
 
         <ModalBody>
           <Form>
             <FormGroup>
-              <FormLabel>Include Unmonitored</FormLabel>
+              <FormLabel>{translate('IncludeUnmonitored')}</FormLabel>
 
               <FormInputGroup
                 type={inputTypes.CHECK}
                 name="unmonitored"
                 value={unmonitored}
-                helpText="Include unmonitored episodes in the iCal feed"
+                helpText={translate('ICalIncludeUnmonitoredHelpText')}
                 onChange={this.onInputChange}
               />
             </FormGroup>
 
             <FormGroup>
-              <FormLabel>Season Premieres Only</FormLabel>
+              <FormLabel>{translate('SeasonPremieresOnly')}</FormLabel>
 
               <FormInputGroup
                 type={inputTypes.CHECK}
                 name="premieresOnly"
                 value={premieresOnly}
-                helpText="Only the first episode in a season will be in the feed"
+                helpText={translate('ICalSeasonPremieresOnlyHelpText')}
                 onChange={this.onInputChange}
               />
             </FormGroup>
 
             <FormGroup>
-              <FormLabel>Show as All-Day Events</FormLabel>
+              <FormLabel>{translate('ShowAsAllDayEvents')}</FormLabel>
```

Comment:
I think we should prefix this with `ICal` same as the help text is.

Reply:
done